build_defs/build_defs.bzl

```diff
@@ -56,5 +56,5 @@ def arcs_ts_test(name, src, deps):
         srcs = [src],
         cmd = "./tools/sigh test --bazel --file {SRC}",
         tags = EXECUTION_REQUIREMENTS_TAGS,
-        deps = deps,
+        deps = deps + ["//src:core"],
```

Comment:
Depending on what you want to achieve, you probably want to add this to some other rules too. Adding this line here will cause bazel to re-run the test whenever Arcs runtime sources change. But it won't re-run other dependencies like arcs_cc_schema for instance. So if you edited schema2pkg.ts and then re-ran your unit test, it wouldn't do the code-gen again.

I think adding this dep to arcs_cc_schema and arcs_kt_schema should be sufficient, you won't need it for cc_wasm_binary (they'll only get rebuilt if the generated cc source changes, which is the behaviour we want)

Reply:
I did think about that, but it seemed a bit heavyweight to make schema2pkg depend on the entire runtime source. I'm willing to live with disabling caching on the rare occasions I'm changing the packager and running tests.

Reply:
Fair enough! You could add another filegroup just for schema2pkg sources if you were so inclined.